src/ipc/bridge.cc

```diff
@@ -2601,7 +2612,14 @@ static void registerSchemeHandler (Router *router) {
     body = (char *) data;
   }
 
+  [_tasks addObject: task];
```

Comment:
Only `router->invoke` tasks are added, as the other possibilities are synchronous if I'm not mistaken.